docs/user/intro.rst

```diff
@@ -0,0 +1,58 @@
+WiFi Login Pages: Features
+==========================
+
+OpenWISP WiFi login pages offers the following features:
+
+- Mobile first design (responsive UI)
+- Sign up
+- Optional support for mobile phone verification: verify phone number by
+  inserting token sent via SMS, resend the SMS token
+- Login to the WiFi service (by getting a radius user token from OpenWISP
+  Radius and sending a POST to the captive portal login URL behind the
+  scenes)
+- Session status information
+- Logout from the WiFi service (by sending a POST to the captive portal
+  logout URL behind the scenes)
+- Change password
+- Reset password (password forgot)
+- Support for :ref:`Social Login <wlp_social_login>` and :ref:`SAML
+  <wlp_saml>`
+- Optional social login buttons (Facebook, Google, X/Twitter)
+- Contact box showing the support email and/or phone number, as well as
+  additional links specified via configuration
+- Navigation menu (header and footer) with the possibility of specifying
+  if links should be shown to every user or only authenticated or
+  unauthenticated users
+- Support for multiple organizations with the possibility of customizing
+  the theme via CSS for each organization
+- Support for multiple languages
+- Possibility to change any text used in the pages
+- Configurable Terms of Services and Privacy Policy for each organization
+- Possibility of automatically logging in users who signed in previously
+  (if the captive portal browser of their operating system supports
+  cookies)
```

Comment:
here please add a reference to https://openwisp.io/docs/tutorials/hotspot.html#automatic-captive-portal-login